script/dev-deploy

```diff
@@ -1253,7 +1265,7 @@ class SimpleService(Service):
         self.GRAFANA_SVC = super().get_service(GrafanaService)
 
     def init_parser(self):
-        self.cmd.add_argument("--config", default=DEFAULT_DEPLOY_CONF)
+        self.cmd.add_argument("--config", default=fall_back_default_conf())
```

Comment:
what if `fall_back_default_conf()` return None? The default value should always be a legal value, otherwise there is no point in setting a default value

Reply:
ignore this